docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/server-default-log-level.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Strapi Server log level configuration
+description: Strapi 5 allows users greater control over the level of detail in of their server logs
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - configuration
+ - middlewares
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Strapi Server log level configuration
```

Comment:
Please update the title (see [this comment](https://github.com/strapi/documentation/pull/2049/files#r1521906638))